drud/ddev
# Create settings.ddev.php on ddev start, fixes #1423

## The Problem/Issue/Bug:

OP #1423: Teams often share a project and don't run `ddev config` individually. So the settings.ddev.php or equivalent needs to be created on `ddev start`

## How this PR Solves The Problem:

Add settings creation into the post-start action.

## Manual Testing Instructions:

* Remove settings.ddev.php
* ddev start
* Verify that it exists again

Unfortunately this needs to be tested in every supported CMS; TYPO3 and Wordpress and Backdrop are the most likely places for unexpected consequences.

## Automated Testing Overview:


## Related Issue Link(s):

## Release/Deployment notes:
